contextbasedrestrictionsv1/context_based_restrictions_v1_examples_test.go

```diff
@@ -132,9 +136,9 @@ var _ = Describe(`contextbasedrestrictionsv1.ContextBasedRestrictionsV1 Examples
 			}
 
 			createZoneOptions := contextBasedRestrictionsService.NewCreateZoneOptions()
-			createZoneOptions.SetName("SDK TEST - an example of zone")
```

Comment:
We probably want to keep the `SDK TEST -` prefix for our own testing.

Reply:
I removed them to use the default one so next time when we run the code generator we will have less thing to update